scripts/functional-tests.sh

```diff
@@ -232,7 +232,10 @@ EOF
     sed -i.bak -e 's/num_instructions:.*/num_instructions: 1/g' config.yaml
 
     # This should be finished in a minut or so but time it out incase it goes wrong
-    timeout 10m ilab generate --taxonomy-path test_taxonomy/compositional_skills/simple_math.yaml
+    if ! timeout 20m ilab generate --taxonomy-path test_taxonomy/compositional_skills/simple_math.yaml; then
```

Comment:
nit: could you fix the text in the above comment while you're in here :smile:

Reply:
Actually, the comment is still valid, most of the time the process takes a few mins.

Reply:
there's a typo - `minut` should be `minute`

Reply:
Ahhhhh the typo, ok ok :)